### PR TITLE
[poc] bridges cf.js into playground-poc for testing

### PR DESCRIPTION
**Pull request by snario**
@mvanderh 

- used `cuid` over `uuid`
- kept dependencies external
- removed lodash entirely

### CODE REVIEW

**Comment by snario** [comment]
I had to use `import * as ethers from "ethers"` b/c ethers does not support `import ethers from "ethers"` apparently (the tests treat it as undefined). Then, `import { ethers } from ethers` also doesn't work because the rollup can't find the subproperty because the browser version doesn't expose `ethers.ethers`.

File: packages/cf.js/src/legacy/network.ts [hunk 1]
```diff
@@ -43,7 +42,7 @@ export class NetworkContext {
 
   public linkedBytecode(unlinkedBytecode: string): string {
     let bytecode = unlinkedBytecode;
-    for (const contractName of _.keys(this.contractToVar)) {
+    for (const contractName of Object.keys(this.contractToVar)) {
```
Comment (mvanderh) [inline packages/cf.js/src/legacy/network.ts]: I think you can even just use `for (const contractName in this.contractToVar)`

File: packages/cf.js/src/legacy/utils/index.ts [hunk 2]
```diff
@@ -31,5 +30,5 @@ export async function mineBlocks(
   n: number,
   provider: ethers.providers.JsonRpcProvider
 ) {
-  _.times(n, async () => await mineOneBlock(provider));
+  [...Array(n)].forEach(async () => await mineOneBlock(provider));
```
Comment (mvanderh) [inline packages/cf.js/src/legacy/utils/index.ts]: `Array(n).forEach` doesn't work?

File: packages/playground-poc/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+```
+python -m SimpleHTTPServer
+```
+Here
+```
+node server
+```
+open
+```
+localhost:8080
```
Comment (mvanderh) [inline packages/playground-poc/README.md]: I think you mean `localhost:8000`

File: packages/playground-poc/public/index.html [hunk 4]
```diff
@@ -20,9 +20,13 @@
     <script src="https://unpkg.com/eventemitter3@latest/umd/eventemitter3.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/vex-js/4.1.0/js/vex.combined.min.js"></script>
     <script src="/socket.io/socket.io.js"></script>
-    <script src="/scripts/mini-node.js"></script>
-    <script src="/scripts/dapp.js"></script>
-    <script src="/scripts/playground.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/cuid/1.3.8/browser-cuid.min.js"></script>
+    <script src="http://localhost:8000/node_modules/ethers/dist/ethers.js"></script>
```
Comment (mvanderh) [inline packages/playground-poc/public/index.html]: Ah so packaging ethers into the IIFE didn't work?

Comment (mvanderh) [reply packages/playground-poc/public/index.html]: 👍

**Pull request merged**